CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 - Allow unique_key to take a list ([#2479](https://github.com/dbt-labs/dbt-core/issues/2479), [#4618](https://github.com/dbt-labs/dbt-core/pull/4618))
 - Add `--quiet` global flag and `print` Jinja function ([#3451](https://github.com/dbt-labs/dbt-core/issues/3451), [#4701](https://github.com/dbt-labs/dbt-core/pull/4701))
 - Add space before justification periods ([#4737](https://github.com/dbt-labs/dbt-core/issues/4737), [#4744](https://github.com/dbt-labs/dbt-core/pull/4744))
+- Add `--no-print` global flag ([#4710](https://github.com/dbt-labs/dbt-core/issues/4710))
```

Comment:
We've updated the process we use to generate changelogs so that you no longer have to manually figure out where they go.  That's also what the cause of the failing GitHub action is.  You'll need to delete this line and then go through the new process.  You can see the details of how to do it in our [Contributing guide](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md#adding-changelog-entry).

Reply:
Thank you @emmyoop! I've updated the changelog following the new process and I've just pushed the changes.